.github/workflows/build-images-base.yaml

```diff
@@ -57,6 +57,12 @@ jobs:
       - name: Cleanup Disk space in runner
         uses: ./.github/actions/disk-cleanup
 
+      - name: Checkout base branch (trusted)
+        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+        with:
+          ref: ${{ github.base_ref }}
+          persist-credentials: false
+
```

Comment:
About 10 lines up, we already check out `default_branch`. What's the difference? Do we need both? Surely we can use the `./.github/action/disk-cleanup` from the same branch..?

Reply:
The default branch is "main" this line is the "stable branch" which can be "main" or "v1.1x". The GH action that we have to load environment variables is load from main thus I figured that the the disk-cleanup should have the same logic.